src/cmd/build.js

```diff
@@ -110,7 +111,9 @@ async function defaultPackageCreator(
 
   await streamToPromise(stream);
 
-  log.info(`Your web extension is ready: ${extensionPath}`);
+  if (showReadyMessage) {
```

Comment:
Thanks! That was surprisingly easy to do! I'll definitely read on that to avoid mishaps